src/pybind/mgr/dashboard/frontend/src/app/ceph/new-dashboard/dashboard/dashboard.component.ts

```diff
@@ -28,12 +30,11 @@ import { SummaryService } from '~/app/shared/services/summary.service';
 export class DashboardComponent implements OnInit, OnDestroy {
   detailsCardData: DashboardDetails = {};
   osdSettings$: Observable<any>;
-  interval: number;
+  interval = new Subscription();
```

Comment:
Are we sure about this? This is immediately overwritten in the ngOnInit.

Reply:
Yeah, I am only creating a new Subscribtion here.